### DISCUSSION TITLE
I want to list all Terms in the Taxonomy

### DISCUSSION THREAD

**Discussion by hlorofos** [statamic/cms#7513] [Q&A] [answered]
I have Taxonomy named Categories and I want to get a list of all Terms in the given Taxonomy.
But it lists only Terms which are connected to some Collection items. I want to list all the Terms
```
{{ categories }}
  <img src="{{ icon:url }}"> <a href="{{ url }}">{{ title }}</a>
{{ /categories }}
```

**Comment by jasonvarga** [comment, marked as answer]
Use the `taxonomy` tag instead of the entry's field (`categories`).

```diff
-{{ categories }}
+{{ taxonomy:categories }}
  <img src="{{ icon:url }}"> <a href="{{ url }}">{{ title }}</a>
-{{ /categories }}
+{{ /taxonomy:categories }}
```

**Comment by hlorofos** [comment]
Awesome @jasonvarga, thank you very much! I haven't realized you support engineers here, good job =)
